### ISSUE TITLE
Ability for the user to see either type or organizer when looking events

### ISSUE THREAD

**Issue by robertpragai** [bettermtl/bettermtl.github.io#50]
It would be good to display the type or organizer on the tile when looking events.
I attached a screenshot for displaying the organizer.
![displaying organizer](https://cloud.githubusercontent.com/assets/15693174/14430470/bee29d5a-ffcf-11e5-992f-c6622dcdf1b3.png)

**Comment by pcuci** [comment]
Yeah, we were also toying with the idea of removing the title altogether in
favour of a few keywords. The organizer here acts like a brand, they
already have too much power/influence/clout. BetterMTL is about bottoms-up
activities for organizers to adapt to what the city wants to confer about.

I'll wait for more people to complain about this before implementing, the
assumption being the organizer is important - it is, but I think the
content/topic/keywords and the people showing up (who've rsvped, including
VIPs) are bigger attractors.

**Comment by David-Ro** [comment]
Wikiinfo, wikifactchecker. 
Wikiinviter, wikiconnector.
Wikiteammaker, wikitracker.
Who 
Where, when, which_admission_conditions
Which interests, subject, topic, material
